### PR TITLE
Cleaned up TS infantry and player structure sequences

### PR DESCRIPTION
**Pull request by reaperrr**

### CODE REVIEW

File: mods/ts/sequences/infantry.yaml [hunk 1]
```diff
@@ -209,99 +185,97 @@ e3:
...
 		ShadowStart: 190
-		Tick: 80
 	die6: electro
 		Length: *
-		Tick: 80
 	icon: e4icon
 
 weedguy:
-	stand: weed
+	Defaults: weed
```
Comment (Phrohdoh) [inline mods/ts/sequences/infantry.yaml]: This makes **everything** weed, even the ones that should probably be `weedguy`, provided that is a valid image.

File: mods/ts/sequences/infantry.yaml [hunk 2]
```diff
@@ -1483,61 +1302,70 @@ doggie: # TODO: not sure what frame 88 and following is
...
+	shoot:
+		Start: 56
+		Length: 4
+		Facings: 8
+		ShadowStart: 175
+	laydown: # TODO: Implement hiding when on a tiberium patch
+		Start: 88
+		Length: 2
+		ShadowStart: 207
+	hide: # TODO: Implement hiding when on a tiberium patch
```
Comment (Phrohdoh) [inline mods/ts/sequences/infantry.yaml]: I'm pretty sure 'hiding' was just a visual bonus as they looked like the Tiberium.
They weren't invisible.

Comment (reaperrr) [inline mods/ts/sequences/infantry.yaml]: That's not what I was implying. They just lay down and look almost like a tiberium patch.

File: mods/ts/sequences/infantry.yaml [hunk 3]
```diff
@@ -1579,23 +1405,37 @@ civ1:
...
 		ShadowStart: 451
 		Tick: 800
 		ZOffset: -511
+	shoot:
+		Start: 164
+		Length: 6
+		Facings: 8
+		ShadowStart: 456
+	prone-shoot:
+		Start: 212
```
Comment (abcdefg30) [inline mods/ts/sequences/infantry.yaml]: Those frames look empty in the assets browser?

**Comment by abcdefg30** [comment]
Looks good to me with above resolved. :+1: / :white_check_mark:

File: mods/ts/sequences/infantry.yaml (hunk 3 above)
Comment (reaperrr) [inline mods/ts/sequences/infantry.yaml]: Bogus copy-pasta from mutants. Fixed, thanks.

**Comment by reaperrr** [comment]
Updated.

Note: I noticed that the ^CivilianInfantry default has a pistol armament even though civ2 art lacks shooting frames, but I'll deal with that in my rules-split pr.

**Comment by pchote** [comment]
Looks good to me, and this only touches the ts yaml so there is no reason to continue delaying this. :+1:

**Pull request merged**